src/bridge/cockpitrouter.c

```diff
@@ -375,11 +397,31 @@ create_channel (CockpitRouter *self,
                 GType type)
 {
   CockpitChannel *local;
+  const gchar *group_name;
+  ChannelGroup *group;
+
+  if (!cockpit_json_get_string (options, "group", "default", &group_name))
+    g_warning ("%s: caller specified invalid 'group' field in open message", channel);
```

Comment:
I know the current code does the same, but I still have to ask: Is it wise to create the channel anyway with the `default` group instead of just failing and not create it at all? This would clearly be a programming error and like this it's just papered  over.